src/metabase/models/database.clj

```diff
@@ -192,7 +192,9 @@
              :cache_field_values_schedule new-fieldvalues-schedule)))))))
 
 (defn- pre-insert [database]
-  (handle-secrets-changes database))
+  (-> database
+   handle-secrets-changes
+   (assoc :initial_sync_status "incomplete")))
```

Comment:
I'm setting this field to `incomplete` here for new DBs because it will otherwise it will default to `complete`. This is because existing databases should have `initial_sync_status` set to `complete` when the migration is run.